martin/src/bin/martin-cp.rs

```diff
@@ -171,7 +171,13 @@ fn compute_tile_ranges(args: &CopyArgs) -> Vec<TileRect> {
                 bbox_to_xyz(bbox.left, bbox.bottom, bbox.right, bbox.top, *zoom);
             append_rect(
                 &mut ranges,
-                TileRect::new(*zoom, min_x, min_y, max_x, max_y),
+                TileRect::new(
+                    *zoom,
+                    u32::try_from(min_x).unwrap(), //todo we should use u64 directily
```

Comment:
Maybe use u64 directly better, maybe next PR?

Reply:
just use `as` and disable the casting warning with `allow`